packages/lab/stories/slider/slider.stories.tsx

```diff
@@ -1,60 +1,247 @@
-import { Button, Card, Checkbox } from "@salt-ds/core";
-import { Slider, type SliderProps } from "@salt-ds/lab";
+import {
+  FlexLayout,
+  FormField,
+  FormFieldHelperText,
+  FormFieldLabel,
+  Input,
+  StackLayout,
+} from "@salt-ds/core";
+import { Slider, type SliderProps, type SliderValue } from "@salt-ds/lab";
 import type { StoryFn } from "@storybook/react";
+import { type ChangeEvent, useEffect, useState } from "react";
+
+const validateSingle = (value: string, bounds: [number, number]) => {
+  if (Number.isNaN(Number(value))) return false;
+  if (Number(value) < bounds[0] || Number(value) > bounds[1]) return false;
+  return true;
+};
+
+const validateRange = (values: [string, string], bounds: [number, number]) => {
+  if (values.length !== 2) return false;
+  const [min, max] = values;
+  const minValid = validateSingle(min, bounds);
+  const maxValid = validateSingle(max, bounds);
+  if (!minValid || !maxValid) return false;
+  if (Number(min) > Number(max)) return false;
+  return true;
+};
 
 export default {
   title: "Lab/Slider",
   component: Slider,
 };
 
-const SliderTemplate: StoryFn<SliderProps> = (args) => {
-  return <Slider {...args} />;
+const Template: StoryFn<SliderProps> = ({ ...args }) => {
+  return <Slider style={{ width: "300px" }} {...args} />;
 };
 
-const SliderOnACardTemplate: StoryFn<SliderProps> = (props) => {
-  return (
-    <Card>
-      <Checkbox label="Coffee" />
-      <Checkbox label="Black Tea" />
-      <Checkbox label="Green Tea" />
-      <Slider label="Sugar" min={0} max={5} defaultValue={1} />
-      <Slider
-        label="Milk"
-        defaultValue={50}
-        marks={[0, 10, 50, { value: 100, label: "100% Milk" }]}
-        {...props}
-      />
-      <Checkbox label="Iced" />
-      <Button>Start</Button>
-    </Card>
-  );
+export const Default = Template.bind({});
+Default.args = {
+  min: 0,
+  max: 10,
+  "aria-label": "default",
 };
 
-export const Simple = SliderTemplate.bind({});
+export const WithNegativeBounds = Template.bind({});
+WithNegativeBounds.args = {
+  min: -5,
+  max: 5,
+  "aria-label": "WithNegativeBounds",
```

Comment:
It's probably fine in most screen readers, but aria labels should really be structured as sentences